src/raft/proto/binlog.proto

```
@@ -0,0 +1,44 @@
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package kiwi.raft.binlog;
```

Buf (1.59.0)

Files with package "kiwi.raft.binlog" must be within a directory "kiwi/raft/binlog" relative to root but were in directory "src/raft/proto".

(PACKAGE_DIRECTORY_MATCH)